code/src/components/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState} from 'react';
+import { MovieThumb } from './MovieThumb';
+
+import "./movieList.css";
+
+export const MovieList = () => {
+    const MOVIES_URL = 'https://api.themoviedb.org/3/movie/popular?api_key=0cb608022c0a0fbef75881f68133d694&language=en-US&page=1';
+    const [movies, setMovies] = useState([]);
+
+    console.log('Render');
```

Comment:
I think it's good practice to remove console logs before pushing to github 😊

Reply:
👍